mtgjson5/price_builder.py

```diff
@@ -113,6 +114,42 @@ def build_today_prices(self) -> Dict[str, Any]:
 
         return final_results
 
+    def build_today_prices_v2(self) -> MtgjsonPricesV2Container:
+        """
+        Build v2 price records by calling build_v2_prices() on each provider.
+
+        Aggregates v2 records from providers that support the v2 interface.
+        Providers without v2 support will return empty lists.
+
+        :return: Container with all v2 price records from today
+        """
+        if not self.all_printings_path.is_file():
+            LOGGER.error(
+                f"Unable to build v2 prices. AllPrintings not found in {MtgjsonConfig().output_path}"
+            )
+            return MtgjsonPricesV2Container()
+
+        v2_container = MtgjsonPricesV2Container()
+
+        for provider in self.providers:
+            provider_name = type(provider).__name__
```

Comment:
this feels suspect at best. I don't like depending on double underscore components if not necessary